packages/hint-scoped-svg-styles/tests/hint.ts

```diff
@@ -0,0 +1,94 @@
+import { test, fs } from '@hint/utils';
+import { HintTest, testHint } from '@hint/utils-tests-helpers';
+
+const { readFile } = fs;
+const { generateHTMLPage, getHintPath } = test;
+const hintPath = getHintPath(__filename);
+
+const generateConfig = (fileName: string) => {
+    if (fileName.endsWith('.html')) {
+        return readFile(`${__dirname}/fixtures/${fileName}`);
+    }
+
+    const styles = readFile(`${__dirname}/fixtures/${fileName}.css`);
+
+    return {
+        '/': generateHTMLPage(
+            `<link rel="stylesheet" href="styles/${fileName}">`
+        ),
+        [`/styles/${fileName}`]: {
+            content: styles,
+            headers: { 'Content-Type': 'text/css' }
+        }
+    };
+};
+
+/*
```

Comment:
I think we don't need this auto-generated message :)

Reply:
removed.